apis/python/examples/ingesting-data-files.md

```diff
@@ -105,7 +105,7 @@ queries benefit from that optimized storage. In particular, various cross-corpus
 in these examples take just seconds or minutes.
 
 A key point is **write once, read from multiple tools** -- in particular, using `tiledbsoma-py` (this
-package) or [`tiledbsoma-r`](https://github.com/TileDB-Inc/tiledbsoma) you can read SOMAs in either
+package) or [`tiledbsoma-r`](https://github.com/TileDB-Inc/TileDB-SOMA) you can read SOMAs in either
```

Comment:
is the repo path wrong?

Reply:
@bkmartinjr i've found & fixed several issues on more recent commits -- thank you!